habitat-lab/habitat/config/benchmark/ovmm/ovmm.yaml

```diff
@@ -68,5 +68,6 @@ habitat:
       enable_physics: True
     needs_markers: false
   dataset:
+    data_path: data/episodes/rearrange/v4/val/cat_npz-exp-with_init_poses.json.gz
```

Comment:
IDK if you have another PR coming up for this but we should move episodes to 
`data/episodes/ovmm/v1/...`

Reply:
Yeah I'll make a PR after these are merged to update the final data locations.